synapse/storage/databases/main/stream.py

```diff
@@ -785,22 +785,14 @@ async def get_last_event_in_room_before_stream_ordering(
         return None
 
     async def get_current_room_stream_token_for_room_id(
-        self, room_id: Optional[str] = None
```

Comment:
I've checked the usages. We never have to worry about a case where the `room_id` is optional.

```
get_current_token_for_pagination
get_current_key_for_room
get_current_room_stream_token_for_room_id
```